dev_mode/package.json

```diff
@@ -207,7 +207,7 @@
     "webpack-merge": "^5.8.0",
     "whatwg-fetch": "^3.0.0",
     "worker-loader": "^3.0.2",
-    "yarn-deduplicate": "^5.0.0"
+    "yarn-berry-deduplicate": "^2.0.0"
```

Comment:
Do we need that dependency ?
It is always used with `dlx` command, which seems to download the package in a temporary environment.

Reply:
Good point - and indeed then it will be good to update the migration guide - I did not because I was trying to avoid adding a tool for all extensions.